lib/onebox/engine/html.rb

```diff
@@ -11,8 +11,8 @@ def http_params
       end
 
       def raw
-        body_cacher = self.options[:body_cacher] if self.options
-        @raw ||= Onebox::Helpers.fetch_html_doc(url, http_params, body_cacher)
+        @body_cacher = self.options[:body_cacher] if self.options
```

Comment:
I think it might be better to extract this into a separate method:

```rb
def body_cacher
  self.options&.[](:body_cacher)
end
```

…and then use that (`self.body_cacher` or just `body_cacher`) instead of `@body_cacher`.

Reply:
You're right - that's a much neater way of doing it.

Thanks for your thoughtful feedback and comments!